tests/test_requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytket-qiskit~=0.23
+pytket-quantinuum
```

Comment:
Can ya update the version to be >= the latest version please? As this will have the fixes in